mustard_ui.py

```diff
@@ -319,6 +321,11 @@ class MustardUI_OptionItem(bpy.types.PropertyGroup):
                         name = "Option value",
                         update = mustardui_body_additional_options_update,
                         description = "Value of the property")
+
+    body_big_float_value: bpy.props.FloatProperty(min=0., max=10.,

```

Comment:
The definition is correct, but the new BigFloat are only stored and not used. To use them we have to add them to the UI, i.e. we have to add these definitions in the following lines:

- [ ] 4828
- [ ] 4848

Reply:
Good catch, update made 👍